.gitignore

```diff
@@ -240,4 +240,20 @@ conda/pkg
 # Downloaded models/datasets
 .tvm_test_data
 .dgl
-.caffe2
\ No newline at end of file
+.caffe2
+
+# logs
+*.txt
+*.xlsx
+*.cover
+txt2excel_profile.py
```

Comment:
we don't need to add those contents in the file as they're more like our own file names for debugging purpose.

Reply:
delete the irrelevant contents.